proto/kvrpcpb.proto

```diff
@@ -176,11 +176,17 @@ message ScanLockResponse {
     repeated LockInfo locks = 3;
 }
 
+message Txn2Status {
```

Comment:
s/Txn2Status/TxnStatus

Reply:
TxnStatus has been used by tidb as an alias for uint64.
So we must use another name.

Reply:
use a better name, not 2.

Reply:
how about TxnInfo?
@siddontang